### DISCUSSION TITLE
Button size customization?

### DISCUSSION THREAD

**Discussion by brendan-Wursta** [rainbow-me/rainbowkit#590] [General]
Library is great, but I don't see setting padding or size for the button in the theme options. Does this simple change require an entirely custom button setup?
https://www.rainbowkit.com/docs/theming

**Comment by nickbytes** [comment]
Hey, thanks for trying out RainbowKit!

> Library is great, but I don't see setting padding or size for the button in the theme options. Does this simple change require an entirely custom button setup? https://www.rainbowkit.com/docs/theming

If you'd like to customize the appearance of the connect button, you have two options:
1. Create a Custom Connect Button (like you noted) https://www.rainbowkit.com/docs/custom-connect-button – this would allow you to customize everything about the appearance and interactions of the Connect Button. This is my favorite option for making RainbowKit look/feel more like your brand.
2. Use CSS to target and override the default styles of the Connect Button.

Our theming API currently doesn't include all possibly styling properties (like padding, heights, etc etc).

Does this answer your question?
